### PR TITLE
Cleanup user view

### PR DESCRIPTION
**Pull request by DanielW1987**
I have removed the "Login Log" tab.
Also, the date and time strings are now formatted according to the browser locale.

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Cleanup User View in admin area](https://trello.com/c/HHW097cf/56-cleanup-user-view-in-admin-area)

File: webapp/src/main/resources/static/css/admin.css [hunk 1]
```diff
@@ -57,6 +57,14 @@ div.content {
   }
 }
 
+.default-modal-dialog-with {
```
Comment (NilsDammenhayn) [inline webapp/src/main/resources/static/css/admin.css:60]:
```suggestion
.default-modal-dialog-width {
```

File: webapp/src/main/resources/static/css/admin.css [hunk 2]
```diff
@@ -57,6 +57,14 @@ div.content {
...
+  max-width: 35rem;
+}
+
+.large-modal-dialog-with {
```
Comment (NilsDammenhayn) [inline webapp/src/main/resources/static/css/admin.css:64]:
```suggestion
.large-modal-dialog-width {
```

File: webapp/src/main/resources/templates/admin/releases/update.html [hunk 3]
```diff
@@ -4,8 +4,8 @@
     <title>Update Release</title>
 </head>
 <body>
-<div th:fragment="dialog" class="modal-dialog" role="document">
-    <div class="modal-content modal-content-releases">
+<div th:fragment="dialog" class="modal-dialog large-modal-dialog-with" role="document">
```
Comment (NilsDammenhayn) [inline webapp/src/main/resources/templates/admin/releases/update.html:7]:
```suggestion
<div th:fragment="dialog" class="modal-dialog large-modal-dialog-width" role="document">
```

File: webapp/src/main/resources/templates/admin/users/create.html [hunk 4]
```diff
@@ -4,7 +4,7 @@
         <title>Create User</title>
     </head>
     <body>
-        <div th:fragment="dialog" class="modal-dialog" role="document">
+        <div th:fragment="dialog" class="modal-dialog default-modal-dialog-with" role="document">
```
Comment (NilsDammenhayn) [inline webapp/src/main/resources/templates/admin/users/create.html:7]:
```suggestion
        <div th:fragment="dialog" class="modal-dialog default-modal-dialog-width" role="document">
```

File: webapp/src/main/resources/templates/admin/users/update.html [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <title>Update User</title>
 </head>
 <body>
-    <div th:fragment="dialog" class="modal-dialog" role="document">
+    <div th:fragment="dialog" class="modal-dialog default-modal-dialog-with" role="document">
```
Comment (NilsDammenhayn) [inline webapp/src/main/resources/templates/admin/users/update.html:7]:
```suggestion
    <div th:fragment="dialog" class="modal-dialog default-modal-dialog-width" role="document">
```

**Pull request merged**